iocage/lib/ioc_start.py

```diff
@@ -341,8 +336,8 @@ def start_network_interface_vnet(self, nic, net_configs, jid):
                             self.start_network_vnet(nic, bridge, membermtu,
                                                     iface, ip, gw, jid)
 
-        except:
-            pass
+        except CalledProcessError as err:
+            return f"ERROR: {err.output.decode('utf-8')}".rstrip()
```

Comment:
I prefer to raise an exception if it's an `ERROR` since to me that implies fatal. I think this would be better served as a `WARNING`.